src/field/pokemon.ts

```diff
@@ -945,6 +945,9 @@ export default abstract class Pokemon extends Phaser.GameObjects.Container {
         if (this.status && this.status.effect === StatusEffect.PARALYSIS) {
           ret >>= 1;
         }
+        if (this.getTag(BattlerTagType.UNBURDEN)) {
```

Comment:
```suggestion
        if (this.getTag(BattlerTagType.UNBURDEN) && !this.scene.getField(true).some(pokemon => pokemon !== this && pokemon.hasAbilityWithAttr(SuppressFieldAbilitiesAbAttr))) {
```
For the Neutralizing Gas interaction (please also add a unit test for neut gas Pokemon entering the field when Unburden has already activated)

Reply:
I'll commit this in, but I am confused why Unburden needs an explicit check for this when abilities like Gorilla Tactics and Slow Start do not.